src/backend/rtlsym.d

```diff
@@ -0,0 +1,179 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1994-1998 by Symantec
+ *              Copyright (c) 2000-2016 by Digital Mars, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     backendlicense.txt
+ * Source:      $(DMDSRC backend/_rtlsym.d)
+ */
+
+module ddmd.backend.rtlsym;
+
+import ddmd.backend.cc : Symbol;
+
+enum RTLSYM
+{
+    throwd,
+    throwc,
+    throwdwarf,
+    monitor_handler,
```

Comment:
So... sorry these are still up to snuff. There seems to be a mix of two conventions here: (a) rawconcatenation, (b) things_with_underscores. Neither is correct. The convention we currently use at least for druntime and phobos is camelCase.